### PR TITLE
Add new redirect application to repo list

### PR DESCRIPTION
**Pull request by lbeaufort**
Partial resolution for https://github.com/fecgov/fec-accounts/issues/313

- Add new redirect application [fec-proxy-redirect](https://github.com/fecgov/fec-proxy-redirect) repo to list in README.md
- Remove unneeded files
- Clean up whitespace
- Small language change

### CODE REVIEW

**Review by jason-upchurch — APPROVED** [review]
Thank you @lbeaufort 👍

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ## Campaign finance for everyone
-The Federal Election Commission (FEC) releases information to the public about money that’s raised and spent in federal elections — that’s elections for US President, Senate, and House of Representatives. 
+The Federal Election Commission (FEC) releases information to the public about money that’s raised and spent in federal elections — that’s elections for US President, Senate, and House of Representatives.
 
 Are you interested in seeing how much money a candidate raised? Or spent? How much debt they took on? Who contributed to their campaign? The FEC is the authoritative source for that information.
 
-The new FEC.gov is a collaboration between [18F](http://18f.gsa.gov) and the FEC. It aims to make campaign finance information more accessible (and understandable) to all users. 
+The new FEC.gov is a collaboration between [18F](http://18f.gsa.gov) and the FEC. It aims to make campaign finance information more accessible (and understandable) to all users.
```
Comment (PaulClark2) [inline README.md:7]: FEC.gov began as a collaboration between [18F](https://18f.gsa.gov) and the [FEC](https://www.fec.gov) in an effort to make campaign finance information more accessible (and understandable) to all users. The FEC continues to improve the site and add functionality.

**Review by PaulClark2 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (PaulClark2) [inline README.md:7]: FEC.gov began as a collaboration between [18F](https://18f.gsa.gov) and the [FEC](https://www.fec.gov) in an effort to make campaign finance information more accessible (and understandable) to all users. The FEC continues to improve the site and add functionality.

Comment (PaulClark2) [reply README.md:7]: “The current version of” could be better. We should not call the site “the new” FEC.gov.

Comment (lbeaufort) [reply README.md:7]: @PaulClark2 agreed! What do you think about: 

>FEC.gov began as a collaboration between 18F and the FEC to make campaign finance information more accessible (and understandable) to all users. The FEC continues to improve the site and add functionality.

Comment (PaulClark2) [reply README.md:7]: @lbeaufort this looks good to me.

**Review by PaulClark2 — APPROVED** [review]
You're suggested edits look good to me.

**Pull request merged**